src/vlinder/make_report.py

```diff
@@ -0,0 +1,234 @@
+            pdf.footer_page(self.name, orientation)
+
+        for input_tables in ["fixed_inputs"]:
+            number_of_iterations = round((len(self.input_dict[input_tables]) / 10) + 0.5)
+            for number_iteration in range(0, number_of_iterations):
+                pdf.add_page()
+                if number_of_iterations > 1:
+                    pdf.chapter_title(
+                        self.make_title(input_tables)
+                        + " "
+                        + str(number_iteration + 1)
+                        + "/"
+                        + str(number_of_iterations),
+                        rgb,
+                    )
+                else:
+                    pdf.chapter_title(self.make_title(input_tables), rgb)
+                pdf.chapter_subtitle(self.make_introduction(input_tables))
+                self.visualize("table", input_tables, save=True, number_iteration=number_iteration)
+                image = cv2.imread(str(self.random_number) + "/table" + input_tables + str(number_iteration) + ".png")
+                max_image, width_image, x_pos = self.determine_position_images(orientation, image)
+                if image.shape[1] > max_image:
+                    pdf.image(
+                        str(self.random_number) + "/table" + input_tables + str(number_iteration) + ".png",
+                        x=25,
+                        y=50,
+                        w=width_image,
+                    )
+                else:
+                    pdf.image(
+                        str(self.random_number) + "/table" + input_tables + str(number_iteration) + ".png",
+                        x=x_pos,
+                        y=50,
+                    )
+                pdf.footer_page(self.name, orientation)
+
+        pdf.add_page()
+        pdf.chapter_title(
+            "The decision maker option '"
+            + self.output_dict[scenario]["highest_weighted_dmo"]
+            + "' has the highest weighted appreciations for scenario: "
+            + scenario,
+            rgb,
+        )
+        self.visualize("barchart", "weighted_appreciations", scenario=scenario, stacked=True, save=True)
+        if orientation == "Portrait":
+            pdf.image(str(self.random_number) + "/figure.png", x=25, y=50, w=150)
+        else:
+            pdf.image(str(self.random_number) + "/figure.png", x=25, y=50, w=250)
+        pdf.footer_page(self.name, orientation)
+        return pdf
+
+    def create_report(self, path, scenario, orientation) -> str:
+        """
+        This function saves the created report out of the function make_slides at the desired location
+        :param scenario: the desired scenario which is used in the visualisations
+        :param path: the desired location where the report is saved
+        :param orientation: the desired orientation for PDF format; there is a choice between Portrait of Landscape
```

Comment:
of -> or
wat is "text where the powerpoint is saved"? bedoel je de path? of bedoel je een success message?

Reply:
oh en het is dus geen ppt meer toch?